src/osvr/Util/LogRegistry.cpp

```diff
@@ -187,7 +205,7 @@ namespace util {
             // File sink - rotates daily
             std::string logDir;
             try {
-                size_t q_size = 1048576; // queue size must be power of 2
+                size_t q_size = 65536; // queue size must be power of 2
```

Comment:
@rpavlik I'm not positive, but my reading of the code suggests it might be the number of messages that can be queued up at one time. I think we could probably drastically reduce this value, but didn't want to be too rash until we've gained a little more experience with it. I'm not sure how full the queue gets in practice.